src/colorizer/loaders/workers/urlLoadWorker.ts

```diff
@@ -86,12 +87,12 @@ async function load(url: string, type: FeatureDataType): Promise<Transfer> {
   }
 
   const { min, max, data } = result;
-
-  // TODO: Also generate textures for the data on the worker thread too?
-  // Would need access to the underlying array buffer to transfer
+  // Cannot directly transfer the DataTexture, since the class methods are not transferred.
+  // Instead, transfer the underlying image and reconstruct it on the main thread.
+  const tex = packDataTexture(data, type);
```

Comment:
Following on from above: might make sense to inline most of what `packDataTexture` is doing here:

```typescript
const [width, height] = getSquarestTextureDimensions(data);
const buffer = padToLength(data, type, 0, length);
// (convert NaN to Infinity - maybe break that out into a utility method too?)
```
... then transfer `{ min, max, width, height, data, buffer }`